languages/ruby/exercises/concept/booleans/.meta/design.md

```diff
@@ -0,0 +1,45 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of nil in Ruby.
```

Comment:
```suggestion
The goal of this exercise is to teach the student the basics of the Concept of "nil" in Ruby.
```

Quoted for the concept of nil, rather than the `nil` value itself, here.